karaf/init/src/main/java/org/apache/brooklyn/launcher/osgi/OsgiLauncher.java

```diff
@@ -199,4 +199,7 @@ public void setLocalBrooklynProperties(String localBrooklynProperties) {
         this.localBrooklynProperties = localBrooklynProperties;
     }
 
+    public void setDeserializingClassRenamesProvider(DeserializingClassRenamesProvider.OsgiConfigLoader val) {
```

Comment:
@geomacy Do I still need the `<property name="deserializingClassRenamesProvider" ref="deserializingClassRenamesProvider" />` ?